packages/react/src/components/form/states.tsx

```diff
@@ -0,0 +1,14 @@
+import clsx from "clsx";
+import * as styles from "./form.css";
+import { AuthenticatingProps, Authenticating as AuthenticatingState } from "./authenticating";
+import { ConfigurationProvider } from "../../context/config-context";
+
+export const Authenticating = (props: AuthenticatingProps) => {
+  return (
+    <div className={clsx("sid-form", styles.form)}>
+      <ConfigurationProvider>
```

Comment:
Have you tried using this in the Console ready? Looking only at the code here I'm pretty sure that this will make passing customisation props impossible.

The Console renders a ConfigurationProvider which it uses to inject the customisation props in the UI. So rendering one here breaks that.

Ideally we'd just expose the regular Authenticating state component and wrap it with required context there.

Reply:
Okay sure, I was trying to make the interface as simple as possible which is why it was baked in here